docs/jiva.md

```diff
@@ -0,0 +1,67 @@
+---
+id: jiva
+title: Jiva Overview
+sidebar_label: Jiva
+---
+
+------
+
+
+
+### Jiva
+
+Jiva has a single container image for both controller and replica. Docker image is available at https://hub.docker.com/r/openebs/jiva/ .  Jiva storage engine is developed with Rancher's LongHorn and gotgt as the base. The entire Jiva engine is written in GO language and runs entirely in the user space. LongHorn controller synchronously replicates the incoming IO to the LongHorn replicas. The replica considers a Linux sparse file as the foundation. It supports thin provisioning, snapshotting, cloning of storage volumes.
```

Comment:
Replace the paragraph starting with "LongHorn controller.." with `Each Jiva Volume comprises of a Jiva Controller (or Target) and a set of Jiva Replicas. The Jiva controller synchronously replicates the incoming IO to the Jiva replicas. The replica considers a Linux sparse file as the foundation. It supports thin provisioning, snapshotting of storage volumes.`

Reply:
Update with commit d0df373